api/v1alpha4/machinehealthcheck_types.go

```diff
@@ -45,6 +45,12 @@ type MachineHealthCheckSpec struct {
 	// +optional
 	MaxUnhealthy *intstr.IntOrString `json:"maxUnhealthy,omitempty"`
 
+	// Any further remediation is only allowed if the number of machines selected by "selector" as not healthy
+	// is within the range of "UnhealthyRange"
```

Comment:
I'd include a comment here that this property takes precedence over `MaxUnhealthy`.